src/main/java/io/vertx/core/http/impl/ClientConnection.java

```diff
@@ -414,7 +414,9 @@ protected synchronized void handleClosed() {
 
     // Connection was closed - call exception handlers for any requests in the pipeline or one being currently written
```

Comment:
yes it will have with the next statement:

```
if (currentRequest != null) {
  currentRequest.handleException(e);
}
```

it was just called twice in this case.